### DISCUSSION TITLE
Is pasting text "show" by design?

### DISCUSSION THREAD

**Discussion by trajano** [helix-editor/helix#10877] [Q&A] [answered]
I pasted text from the clipboard on Windows terminal and I notice it to render pretty slowly, it looks pretty, so I am not sure if it was the intention.  If so can that behaviour be disabled?

**Comment by kirawi** [comment]
Use `space-p` to paste.

**Reply by trajano** [reply]
Is it possible to just make it the default?

**Reply by kirawi** [reply, marked as answer]
Windows Terminal has issues with bracketed paste with respect to the upstream library we use https://github.com/crossterm-rs/crossterm/issues/737 (which is a special mode of paste that works like you'd expect; normally "paste" just executes the key sequence in the text rather than actually pasting). There's nothing we can do on our end.
